play-scala-tls-example/play

```diff
@@ -22,40 +22,34 @@ JVM_OPTIONS="$JVM_OPTIONS -Dplay.http.sslengineprovider=https.CustomSSLEnginePro
 #JVM_OPTIONS="$JVM_OPTIONS -Dplay.ssl.needClientAuth=true"
 
 # Enable the handshake parameter to be extended for better protection.
-# http://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSSERefGuide.html#customizing_dh_keys
+# https://docs.oracle.com/en/java/javase/11/security/java-secure-socket-extension-jsse-reference-guide.html#GUID-D9B216E8-3EFC-4882-B76E-17A87D8F2F9D
 # Only relevant for "DHE_RSA", "DHE_DSS", "DH_ANON" algorithms, in ServerHandshaker.java.
 JVM_OPTIONS="$JVM_OPTIONS -Djdk.tls.ephemeralDHKeySize=2048"
 
 # Don't allow client to dictate terms - this can also be used for DoS attacks.
-# Undocumented, defined in sun.security.ssl.Handshaker.java:205
+# Undocumented, defined in https://github.com/openjdk/jdk11u/blob/jdk-11.0.20%2B1/src/java.base/share/classes/sun/security/ssl/ServerHandshakeContext.java#L35-L43
```

Comment:
In Java 8: https://github.com/frohoff/jdk8u-dev-jdk/blob/master/src/share/classes/sun/security/ssl/Handshaker.java#L197-L205
In Java 11: https://github.com/openjdk/jdk11u/blob/jdk-11.0.20%2B1/src/java.base/share/classes/sun/security/ssl/ServerHandshakeContext.java#L35-L43